property_table.py

```diff
@@ -737,6 +736,86 @@ class PropertyTable:
             "FMantissa9",
             False,
         ),
+        "Tgas_cy_weighted": (
+            "GasComptonYTemperature",
```

Comment:
I think this is supposed to be the output HDF5 dataset name, but GasComptonYTemperature appears for several different quantities. Possible copy/paste error?

Reply:
I think I've fixed this. Maybe Joey or Roi could take a look in property_table.py to check that the names I've given the datasets in 264a3a0 are reasonable?

Reply:
Those look good! Thanks